pom.xml

```diff
@@ -45,6 +45,7 @@
     <git.version>3.8.0</git.version>
     <access-modifier-checker.failOnError>true</access-modifier-checker.failOnError>
     <workflow-support.version>3.2</workflow-support.version>
+    <pipeline-model-definition.version>1.3.7</pipeline-model-definition.version>
```

Comment:
Moved to a property to make it easy to keep everything in sync, and the update is required for the JENKINS-43292 regression tests to work.